docs/src/pages/components/authenticator/signUpAttributes.all.angular.mdx

```diff
@@ -0,0 +1,21 @@
+```html{1}
+  <amplify-authenticator [signUpAttributes]="[
+    'address',
+    'birthdate',
+    'email',
+    'family_name',
+    'gender',
+    'given_name',
+    'locale',
+    'middle_name',
+    'name',
+    'nickname',
+    'phone_number',
+    'picture',
+    'preferred_username',
+    'profile',
+    'updated_at',
+    'website',
+    'zoneinfo', ]"
+    ></amplify-authenticator>
```

Comment:
Minor style nit: we can deindent this and match the prettier style.

```suggestion
<amplify-authenticator 
  [signUpAttributes]="[
    'address',
    'birthdate',
    'email',
    'family_name',
    'gender',
    'given_name',
    'locale',
    'middle_name',
    'name',
    'nickname',
    'phone_number',
    'picture',
    'preferred_username',
    'profile',
    'updated_at',
    'website',
    'zoneinfo', 
  ]"
></amplify-authenticator>

```

Reply:
OK I see. I'll get this fix in my next PR.